scripts/install-mender.sh

```diff
@@ -326,7 +326,109 @@ do_install_missing_monitor_dirs () {
     fi
 }
 
+command_exists() {
+    command -v "$@" > /dev/null 2>&1
+}
+
+# Set the LSB_DIST and DIST_VERSION variables guessing the distribution and version;
+# It also checks if this is a forked Linux distro.
+# Credits: https://get.docker.com/
+check_dist_and_version() {
+    # Every system that we officially support has /etc/os-release
+    if [ -r /etc/os-release ]; then
+        LSB_DIST="$(. /etc/os-release && echo "$ID" | tr '[:upper:]' '[:lower:]')"
+    fi
+    case "$LSB_DIST" in
+        ubuntu)
+            if command_exists lsb_release; then
+                DIST_VERSION="$(lsb_release --codename | cut -f2)"
+            fi
+            if [ -z "$DIST_VERSION" ] && [ -r /etc/lsb-release ]; then
+                DIST_VERSION="$(. /etc/lsb-release && echo "$DISTRIB_CODENAME")"
+            fi
+            case "$DIST_VERSION" in
+                focal)
+                    DIST_VERSION="focal"
```

Comment:
Nit:
No-op

Reply:
I know, this is mainly for consistency with the Debian's case and to make the code clear.